client/knowledgebase/containers/App.js

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
+import { App } from '../components';
+
+const mapStateToProps = (state, ownProps) => ({
+  ...ownProps,
+});
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
Remove `mapDispatchToProps`. It's empty.

Reply:
done